bravado/response.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+
+class BravadoResponse(object):
+    def __init__(self, incoming_response, result):
+        self.incoming_response = incoming_response
+        self.result = result
+        self.response_metadata = BravadoResponseMetadata(
```

Comment:
Let's rename this to "metadata" since we're already in the context of "BravadoResponse"

Reply:
Sounds good, I've run into this as well. I'll rename it in an upcoming PR.